composer.json

```diff
@@ -12,21 +12,21 @@
     }
   ],
   "require": {
-    "php": ">=7.4",
+    "php": ">=8.1",
     "ext-curl": "*",
-    "guzzlehttp/guzzle": "^6.3||^7.0",
-    "symfony/config" : "^4.4||^5.0||^6.0",
-    "symfony/dependency-injection": "^4.4||^5.0||^6.0",
-    "symfony/event-dispatcher": "^4.4||^5.0||^6.0",
-    "symfony/http-kernel": "^4.4||^5.0||^6.0",
-    "symfony/yaml" : "^4.4||^5.0||^6.0"
+    "guzzlehttp/guzzle": "^7",
+    "symfony/config" : "^5.4||^6.0",
```

Comment:
ℹ️ [Symfony 4.4 is still supported by Symfony](https://symfony.com/releases/4.4) ("security fixes only") until November... but it should be fine to release this before November 👍

Reply:
As PHP8.0... but I guessed it was the time to make some cleanup. :man_shrugging: